src/main/kotlin/org/rust/ide/annotator/fixes/ChangeFunctionSignatureFix.kt

```diff
@@ -0,0 +1,200 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.cargo.project.workspace.PackageOrigin
+import org.rust.ide.annotator.getFunctionCallContext
+import org.rust.ide.presentation.render
+import org.rust.ide.presentation.renderInsertionSafe
+import org.rust.ide.refactoring.changeSignature.Parameter
+import org.rust.ide.refactoring.changeSignature.ParameterType
+import org.rust.ide.refactoring.changeSignature.RsChangeFunctionSignatureConfig
+import org.rust.ide.refactoring.changeSignature.runChangeSignatureRefactoring
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.containingCrate
+import org.rust.lang.core.psi.ext.isMethod
+import org.rust.lang.core.psi.ext.valueParameters
+import org.rust.lang.core.types.implLookup
+import org.rust.lang.core.types.ty.TyUnknown
+import org.rust.lang.core.types.type
+
+/**
+ * Requires that the number of arguments is larger than the number of parameters.
+ */
+class ChangeFunctionSignatureFix private constructor(
+    arguments: RsValueArgumentList,
+    function: RsFunction,
+    private val direction: ArgumentScanDirection
+) : LocalQuickFixAndIntentionActionOnPsiElement(arguments) {
+    private val fixText = run {
+        val signature = calculateSignature(function, arguments, direction)
+        val argumentInsertions = signature.withIndex().filter { (_, item) -> item is SignatureMember.Argument }
+
+        val callableType = if (function.isMethod) "method" else "function"
+        val name = function.name
+        if (argumentInsertions.size == 1) {
+            val index = argumentInsertions[0].index + 1
+            val argument = argumentInsertions[0].value as SignatureMember.Argument
+
+            val ordinal = "${index}${suffix(index)}"
+            "Add `${argument.expr.type.renderInsertionSafe()}` as `$ordinal` parameter to $callableType `$name`"
+        } else {
+            val signatureText = signature.joinToString(", ") {
+                when (it) {
+                    is SignatureMember.Argument -> "<b>${it.expr.type.renderInsertionSafe()}</b>"
```

Comment:
For some reason `<b>` does not work

![image](https://user-images.githubusercontent.com/6505554/109811437-e281bc80-7c3b-11eb-9278-d461d727ce32.png)

It is interesting how Java plugin does this